app/src/common/shared/org/mozilla/vrbrowser/VRBrowserActivity.java

```diff
@@ -387,6 +391,7 @@ public void onWindowVideoAvailabilityChanged(@NonNull WindowWidget aWindow) {
             whatsNew.setLoginOrigin(Accounts.LoginOrigin.NONE);
             whatsNew.getPlacement().parentHandle = mWindows.getFocusedWindow().getHandle();
             whatsNew.show(UIWidget.REQUEST_FOCUS);
+            mWhatsNewWidget = whatsNew;
```

Comment:
sorry, I just noticed this, replace the local `whatsNew` variable with `mWhatsNewWidget`
```
        if (!SettingsStore.getInstance(this).isWhatsNewDisplayed()) {
            mWhatsNewWidget = new WhatsNewWidget(this);
            mWhatsNewWidget.setLoginOrigin(Accounts.LoginOrigin.NONE);
            mWhatsNewWidget.getPlacement().parentHandle = mWindows.getFocusedWindow().getHandle();
            mWhatsNewWidget.show(UIWidget.REQUEST_FOCUS);
        }
```